Hestia/Hestia/Resources/strings.Designer.cs

```diff
@@ -60,598 +47,436 @@ public class strings {
             }
         }
         
-        /// <summary>
```

Comment:
We are all summaries removed from this file?

Reply:
I didn't do it. This is auto generated, maybe it is different in Windows and Mac